test/ctia/test_helpers/search.clj

```diff
@@ -152,7 +153,7 @@
       (is (empty? (search-ids app entity "wor"))))
 
     ;; test stop word filtering
-    (is (= matched-ids (search-ids app entity "the word"))
+    (is (= matched-ids (search-ids app entity "the word" {:search_fields ["description"]}))
```

Comment:
this test is supposed to work as before.

Reply:
I'm having to keep it for now. Some tests failed when I removed it.

Reply:
If it still fail, there is a bug here since the searchable-fields shall be injected with `description`.